config/my_vars.yml.example

```diff
@@ -39,7 +42,23 @@ dockerhub_org: ansibleplaybookbundle
 # Disable SSL.
 # use_ssl: false
 
+
+##########################################
+# Use Staged Images for RCM
+##########################################
+## Override and use latest images pushed to developer servers to mimic stage
+#
+# broker_registry_url: http://registry.access.stage.redhat.com
+# origin_image_name: registry.ops.openshift.com/openshift3/ose
+#
+# - Dev Staging
+# broker_registry_url: http://asb-registry.usersys.redhat.com:5000
+# origin_image_name: asb-registry.usersys.redhat.com:5000/openshift3/ose
+
+
```

Comment:
What do you think about leaving the rcm specifics in rcm.yml?

so removing lines ~45-58?

Reply:
sure